src/Apps/Auctions/Components/WorksByArtistsYouFollowRail.tsx

```diff
@@ -29,15 +29,19 @@ const WorksByArtistsYouFollowRail: React.FC<WorksByArtistsYouFollowRailProps> =
 
   return (
     <Rail
-      title="Works for you"
-      subTitle="Works at auction by artists you follow"
+      title="Auction Lots for you ending soon"
+      viewAllLabel="View All"
+      viewAllHref="/" //TODO: to be added
```

Comment:
**suggestion:** Would this now navigate the user to the home screen? If so I would remove this line to avoid having issues with the back button

Reply:
Ah right, it's not behind a feature flag, I'll remove it

Reply:
Will add the link in another PR